packages/harmony/.storybook/preview.ts

```diff
@@ -0,0 +1,13 @@
+// This file is used to configure all stories
+import 'assets/styles/fonts.css'
+import 'assets/styles/colors.css'
+import 'assets/styles/tokens.css'
+// import 'assets/styles/sizes.css'
+// import 'assets/styles/animations.css'
+import 'react-perfect-scrollbar/dist/css/styles.css'
```

Comment:
oh interesting, where is this used?

Reply:
came from stems. its not used yet, but is used for the scrollbar component